shotover-proxy/tests/cassandra_int_tests/mod.rs

```diff
@@ -148,6 +150,7 @@ async fn test_cluster_single_rack_v3(#[case] driver: CassandraDriver) {
         };
         standard_test_suite(&connection, driver).await;
         cluster_single_rack_v3::test_dummy_peers(&connection().await).await;
+        routing::test("127.0.0.1", 9042, "172.16.1.2,172.16.1.3,172.16.1.4", 9042).await;
```

Comment:
Is there a reason we need to provide every node in the cluster to scylla?
I would have thought it would derive the topology as other drivers do?
If we specify just the one node we can drop the string parsing for enable_schema_awaiter.

Reply:
I assumed that we might have to wait for the topology info to update in the driver but it seems that was wrong.